### PR TITLE
jsm_documentation.md: Add jsm public doc

### PR DESCRIPTION
**Pull request by IgorBagnucki**
Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

File: docs/osf-trolling-list/jsm_documentation.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Your request is created and visible for us, but we didn't start working on it yet.
+
+### Analysis
+
+Our team is analyzing your request. We will soon let you know what will be the cost of resolving
+this request.
+
+### Waiting for client
+
+Your action is required to continue working on your request.
```
Comment (BeataZdunczyk) [inline docs/osf-trolling-list/jsm_documentation.md:56]: I suggest adding information on what situation we will be waiting for client response - e.g. after our comment with SP estimation we will ask for acceptance, to start the implementation or after we ask for a clarification from Client.

**Review by BeataZdunczyk — COMMENTED** [review]
Please adress also the internal review

File: docs/osf-trolling-list/jsm_documentation.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+### Raising requests using the portal
+
+To add a new request using the web service:
+
+1. Go to [portal](https://jira-3mdeb.atlassian.net/servicedesk/customer/portals) and select the
+   project.
+2. Select a type of request and provide a required data. You can also add attachements by drag
+   and dropping into any part of the web page, or by clicking the `Browse` button.
+3. Click send. Your request is created and awaits our response.
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md]: Changed

File: docs/osf-trolling-list/jsm_documentation.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+![](../images/jsm_add_request.png)
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md:33]: My name is now replaced with `John Smith`

File: docs/osf-trolling-list/jsm_documentation.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+![](../images/jsm_view_requests.png)
+
+### Raising requests using email
+
+If you prefer raising a request using email, you can do this by sending a message to special
+email address generated for the project.
+
+The email address is be provided to you by separate message.
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md]: Fixed

File: docs/osf-trolling-list/jsm_documentation.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+### Commenting a request and adding files
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md]: Title changed

File: docs/osf-trolling-list/jsm_documentation.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+You can always add a comment to the request and attach files. You can do it directly from the
+JSM request page or using email simply by replying to the email thread.
+
+You'll also recieve an information, w
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md]: I have removed the non-finished sentence and added the information suggested by you.

File: docs/osf-trolling-list/jsm_documentation.md (hunk 1 above)
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md]: I have added this information

File: docs/osf-trolling-list/jsm_documentation.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+## Dasharo Support Package
+
+If you are interested in starting cooperation using the Support Package, it is the perfect time
+to start. The Support Package will help us allocate resources and a secure timeline for your
+needs. To ensure that we can provide comprehensive assistance with your current products and
+future new developments, we encourage you to choose our new Support Package for more streamlined
+cooperation, guaranteed availability, and quick, on-time delivery.
+
+## How to add a request
+
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/osf-trolling-list/jsm_documentation.md:22]: I have added a short intro

**Review by artur-rs — APPROVED** [review]

**Pull request merged**